classes/cluster/mk22_lab_dvr/stacklight/init.yml

```diff
@@ -0,0 +1,47 @@
+parameters:
+  _param:
+    heka_elasticsearch_host: ${_param:stacklight_monitor_address}
+    heka_influxdb_host: ${_param:stacklight_monitor_node01_address}
+    heka_aggregator_host: ${_param:stacklight_monitor_address}
+    aggregator_port: 5565
+    grafana_user: admin
+    grafana_password: password
+    grafana_influxdb_host: ${_param:stacklight_monitor_node01_address}
+    elasticsearch_port: 9200
+    influxdb_stacklight_password: lmapass
+    influxdb_admin_password: password
+    influxdb_port: 8086
+    influxdb_database: lma
+    influxdb_user: lma
+    influxdb_password: lmapass
+    nagios_host: ${_param:stacklight_monitor_address}
+    nagios_status_port: 8001
+    nagios_username: nagiosadmin
+    nagios_password: secret
+    nagios_notification_smtp_server: 127.0.0.1
+    nagios_notification_from: 'nagios@localhost'
+    nagios_notification_email: 'root@localhost'
+    stacklight_environment: ${_param:cluster_domain}_prd
```

Comment:
I'd recommend not making a special case for `mk22_lab_dvr` and setting `stacklight_environment` to `${_param:cluster_domain}`. See https://github.com/Mirantis/mk-lab-salt-model/pull/209.